solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -2158,7 +2158,8 @@ public static File getFile(String name) {
       return file;
     }
     throw new RuntimeException(
-        "Cannot find resource in classpath or in file-system (relative to CWD): " + name);
+        "Cannot find resource in classpath or in file-system (relative to CWD): "
```

Comment:
shouldn't the exception have the name as-given.

Reply:
I think it was much easier for me to debug whatever error was happening when I changed to this. I'm not opposed to put it back.